packages/vc.js/package.json

```diff
@@ -41,6 +41,7 @@
     "@transmute/ed25519-signature-2018": "^0.7.0-unstable.77",
     "@transmute/jose-ld": "^0.7.0-unstable.77",
     "@transmute/revocation-list-context": "^0.7.0-unstable.77",
+    "@transmute/secp256k1-key-pair": "^0.7.0-unstable.77",
```

Comment:
why?

Reply:
seems ok as dev dep.

Reply:
Good catch, I was using it at one point as a dev dependency to generate the key, but I don't need to anymore since the key is a fixture now.  I'll take this out